### PR TITLE
Article submission: Ethereum’s response to EIP0 Results: Governance & Community

### PR DESCRIPTION
**Pull request by pet3r-pan**
# New content for Aragon Monthly

_Be sure to check out the [Guide for submitting a new Pull Request](https://monthly.aragon.one/guides/guide_for_submitting_a_new_pull_request/) before submitting your PR_

### Content category
_Article_

#### Content type:
_Opinion Piece_ 

### Title
Ethereum’s response to EIP0 Results: Governance & Community

### Description
A review of the community responses to the EIP0 Survey

### CODE REVIEW

**Comment by Smokyish** [comment]
@pet3r-pan Thanks for your submission! Waiting for the reviewers to check the submission!

File: docs/articles/opinion/Ethereum’s response to EIP0 Results Part 1: Governance & Community.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+*Art from the Song Dynasty — China’s age of invention*
+
+**// The first few paragraphs is just context, skip down to the next big header for the meat of the article.**
+
+It has been three years since the launch of Ethereum and it has taken the center stage of the global tech community. Decisions over the future of Ethereum now reverberate louder than ever and it was a matter of time before we all had to address how we govern ourselves. In April this year, EIP999 was created and the community was once again reminded of the DAO hardfork. Contentious and without finality: Was the DAO hardfork the right decision? Should Parity be allowed to amend the blockchain and recover the funds?
+
+It sparked an awareness and in response to this, [EIP0](https://ethereum-magicians.org/t/eip0-governance-summit-ahead-of-edcon-in-toronto/251) was born. Held days before the start of EDCON 2018, it was a three day invitational hackathon which looked to discover Ethereum could be better governed. Three days passed and little tangible progress was made. But nonetheless, the important first step was taken. With more questions than answers, we all became aware of the mountain in front of us all.
```
Comment (john-light) [inline docs/articles/opinion/Ethereum’s response to EIP0 Results Part 1: Governance & Community.md]: EIP0 -> EIP0 Summit

Comment (john-light) [reply docs/articles/opinion/Ethereum’s response to EIP0 Results Part 1: Governance & Community.md]: > which looked to discover Ethereum could

* which looked to discover how Ethereum could...

File: docs/articles/opinion/Ethereum’s response to EIP0 Results Part 1: Governance & Community.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+## 1. How do we govern Ethereum?
+
+While not a surprise, governance was on the top of the proverbial list. With nearly over a third of people believing Ethereum to be failing at governance, we had replies about how Ethreum failed to learn from the DAO hardkfork and how there was little clarity and order around the current governance process.
+
+> “…the post-parity hack discussions were kind of a mess. Everyone seemed to be talking over each other, rather than at each other. No one seemed to understand each other’s perspectives”
+
+Some called for greater consideration when handling governance. While it is too easy to say “don’t make it personal”, the matter of fact is that when you are dealing with personal reputation, egos, other peoples money and the core values of others, it becomes a complicated matter. “Empathy is key” Others believed that our mistakes would come back to haunt us all. Many believed that how events such as the DAO hack and parity hack were handled disappointingly.
```
Comment (john-light) [inline docs/articles/opinion/Ethereum’s response to EIP0 Results Part 1: Governance & Community.md]: peoples -> people's

Comment (john-light) [reply docs/articles/opinion/Ethereum’s response to EIP0 Results Part 1: Governance & Community.md]: parity -> Parity

File: docs/articles/opinion/Ethereum’s response to EIP0 Results Part 1: Governance & Community.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
...initely. Do we have a solution? No. But I think we can do a lot to make the process of finding one smoother. We need to be considerate of the fact that in most governance cases, SOMEONE or a group will need to make a decision or as the CEO of Uber says ‘the D’. Good calls will be made but so will bad ones. That sourness is inherently unavoidable yet we pretend to live in a world without disappointment. Many times the person you are twitter flaming is just as passionate as the problem as you are and simply wants the best outcome. Instead of the uncontrollable ‘right’ and ‘wrong’, we should be focusing on making better decisions over time. With so much at stake, it is like Poker. You must play without emotion but with belief and aggression. To be a winning player, you must make the right call at all times without baggage from previous failures or fear of future ones. However in order to do that, we must be led by values and a shared vision. Hard and controversial decisions like EIP999 won’t be the last difficult one that Ethereum has to deal with. And when the darkest hours of the night come, we can still depend on that origin mission that united everyone together in the first place.
```
Comment (john-light) [inline docs/articles/opinion/Ethereum’s response to EIP0 Results Part 1: Governance & Community.md]: > passionate as the problem 

passionate about the problem

**Comment by Smokyish** [comment]
@pet3r-pan Thanks for your submission again! Please review the comments @john-light left for considerations into your future submissions.

Will approve this submission as it is now and merge this article.

**Pull request merged**